data/json/professions.json

```diff
@@ -1671,11 +1745,11 @@
     "skills": [ { "level": 4, "name": "speech" } ],
     "items": {
       "both": {
-        "items": [ "suit", "knit_scarf", "socks", "dress_shoes", "wristwatch" ],
+        "items": [ "suit", "knit_scarf", "socks", "dress_shoes", "wristwatch", "gold_watch", "money_strap_fifty" ],
         "entries": [ { "group": "charged_smart_phone" } ]
       },
-      "male": [ "briefs" ],
-      "female": [ "bra", "panties" ]
+      "male": [ "briefs", "gold_dental_grill" ],
```

Comment:
Cufflinks or jewelry might be more appropriate than the dental grill in this case.

Reply:
Done!